test/module/irohad/ordering/on_demand_os_test.cpp

```diff
@@ -204,11 +204,11 @@ TEST_F(OnDemandOsTest, Erase) {
   ASSERT_TRUE(os->onRequestProposal(
       {commit_round.block_round + 1, commit_round.reject_round}));
 
-  for (auto i = commit_round.reject_round;
-       i < commit_round.reject_round + proposal_limit;
+  for (auto i = commit_round.reject_round + 1;
+       i < commit_round.reject_round + 1 + proposal_limit + 2;
```

Comment:
Maybe better to add an explanation for the `commit_round.reject_round + 1 + proposal_limit + 2` expression?